mongodb/userRepository.go

```diff
@@ -17,9 +19,51 @@ func newUserRepository(database *mgo.Database) *UserRepository {
 }
 
 func (u *UserRepository) Store(user *eduboard.User) error {
```

Comment:
This might need to be updated, as this is shared for inserting a new or updating an existing user.